client/lib/abtest/active-tests.js

```diff
@@ -150,4 +150,13 @@ export default {
 		},
 		defaultVariation: 'original',
 	},
+	prefillDomainStepValue: {
+		datestamp: '20190821',
```

Comment:
Maybe update the date here before the test goes live.

Reply:
Updated to yesterday to make sure it's recent but still current for all timezones.